doc/rados/operations/monitoring.rst

```diff
@@ -738,3 +738,39 @@ Print active connections and their TCP round trip time and retransmission counte
 
 	248     89      1       mgr.0   863     1677    0
 	3       86      2       mon.0   230     278     0
+
+Tracking Data Availability Score of a Cluster
+=============================================
+
+Ceph internally tracks the data availability of each pool in a cluster.
+To check the data availability score of each pool in a cluster, 
+the following command can be invoked: 
+
+
+.. prompt:: bash $
+
+   ceph osd pool availability-status
+
+If the cluster has 4 pools, this is what the ``availability-status`` 
+will report:  
+
+.. prompt:: bash $
+
+   POOL       	UPTIME  DOWNTIME  NUMFAILURES  MTBF  MTTR  SCORE 	AVAILABLE
+   rbd            	2m   	21s        	1	2m   21s  0.888889      	1
+   .mgr          	86s    	0s        	0	0s	0s     	1      	1
+   cephfs.a.meta 	77s    	0s        	0	0s	0s     	1      	1
+   cephfs.a.data 	76s    	0s        	0	0s	0s     	1      	1
+
+We consider a pool unavailable if there is potentially any data loss. 
+This means, if there are any PG in the pool not in 
+active state or if there are unfound objects, some data might be
+either unreachable or lost. In such cases, we mark the pool as 
+unavailable. Otherwise the pool is considered available. 
+For example: A pool will be marked available even if an OSD is down 
```

Comment:
Added an example to boost clarity such that users do not assume that any and all failures will be tracked by this feature. Please let me know if this should be removed/worded differently.

Reply:
>We consider a pool unavailable if there is potentially any data loss.
I think this connotes a more dire situation than might exist, it's good to differentiate between _unavailable_ data which can be brought back and _lost_ data that is gone forever.  Suggest something like

A pool is considered ``unavailable`` if data may be unavailable or lost.

Similarly double backticks around available/unavailable elsewhere where they are the literal states reported.

And s/4 pools/four pools/ please

Reply:
Ah, yes, addressed it!